src/js/editor/editor.js

```diff
@@ -754,8 +754,7 @@ class Editor {
     let pastedPost = parsePostFromPaste(event, this.builder, this._parserPlugins);
 
     this.run(postEditor => {
-      let nextPosition = postEditor.insertPost(position, pastedPost);
-      postEditor.setRange(new Range(nextPosition));
```

Comment:
I'm not sure what you mean by "outside", but this can still use the position-returning semantics...I switched it back to the way it was. `postEditor.insertPost` now returns a `Position`